draft-ietf-tls-esni.md

```diff
@@ -1445,9 +1445,13 @@ following capabilities:
 1. The attacker does not know the ECHConfigList used by the server.
 1. The attacker keeps per-connection state only. In particular, it does not
    track endpoints across connections.
-1. ECH and GREASE ECH are designed so that the following features do not vary:
-   the code points of extensions negotiated in the clear; the length of
-   messages; and the values of plaintext alert messages.
+
+Moreover, real ECH and GREASE ECH are designed so that the following features
+do not vary:
+
+1. the code points of extensions negotiated in the clear;
+1. the length of messages; and
```

Comment:
This is a little interesting. The length of the ECH ciphertext only matches if the implementation accurately simulates things. The length of the encrypted certificate only matches if implementation applies the same padding between the two cases.

Not sure how best to phrase this. It's not out-of-scope per se in that we don't completely ignore it. But we also can't claim they were designed not to not vary without any qualifications.

Reply:
This is existing text (just formatted differently), so I'd be inclined to address that in a followup PR or do nothing.

Reply:
What about qualifying "do not very"? "do not very significantly"?

Question: Do we believe it's possible for an implementation to make real ECH computationally indistinguishable from grease ECH? Would such an implementation be non-compliant?

Reply:
I took the suggestion to qualify "do not vary" -- @cjpatton please have a look.

Reply:
I think it's fine as is, but my suggestion was to tack on "significantly". See suggestion above.